### PR TITLE
Test1

### PR DESCRIPTION
**Pull request by flagxor**
Seth, not sure if this illuminates the format as well as I hoped.
Open to formatting or other suggestions.

### CODE REVIEW

File: hello.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# Hello Wasm
```
Comment (jfbastien) [inline hello.md]: WebAssembly

File: hello.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Production C/C++ compilers like Emscripten/LLVM produce WebAssembly
+with extensive external bindings and standard libraries.
+When all of that is stripped away, a simple hello world program
+looks something like this:
+
+```
```
Comment (jfbastien) [inline hello.md]: For those not used to objdump, can you put a comment at the top explaining each column?

File: hello.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+000007c: 12                                        ; data segment size
+; data segment data 0
+000007d: 4865 6c6c 6f20 5765 6241 7373 656d 626c   ; "Hello WebAssembly\n"
+000008d: 790a                                      ; data segment data
+
+```
+
+The original C program:
+
+```
```
Comment (jfbastien) [inline hello.md]: You can put `cpp` after the triple backtick to get syntax highlight. Same above, you can use `objdump`.

File: hello.md (hunk 1 above)
Comment (flagxor) [inline hello.md]: done

**Comment by flagxor** [comment]
PTAL
Seth, use if you feel it's useful.

**Comment by s3ththompson** [comment]
Closing this one for now.

**Pull request closed without merging**